scripts/oecophylla.py

```diff
@@ -4,11 +4,89 @@
 def run():
     pass
 
+
+def _arg_split(ctx, param, value):
+    # split columns by ',' and remove whitespace
+    _files = [c.strip() for c in value.split(',')]
+    return _files
+
+@run.command()
+@click.option('--input-dir', required=True, type=click.STRING,
+              callback=_arg_split,
+              help='Input directories of all of the samples.')
+@click.option('--sample-sheet', required=True, type=click.STRING,
+              callback=_arg_split,
+              help='Sample sheets used to demultiplex the illumina run.')
+@click.option('--params', type=click.PATH, required=True,
+              help='Specify parameters for the tools.')
+@click.option('--local-scratch', type=click.PATH, required=False,
+              help='Temporary directory for storing intermediate files.')
+@click.option('--log-dir', type=click.PATH, required=False,
+              help='Directory containing log files.')
+@click.option('--restart', is_flag=True, default=False,
+              help='Restarts the run and overwrites previous input.')
+@click.option('--output-dir',
+              help='Input directory of all of the samples.')
+# TODO: extra snakemake parameters with defaults
+
+# TODO: everything past here will be passed directly into snakemake
+def local_workflow():
+    """
```

Comment:
could you write a clearer description? this docstring is the help message exposed by `click` to end users